queue_services/entity-bn/src/entity_bn/bn_processors/__init__.py

```diff
@@ -87,7 +87,7 @@ def build_input_xml(template_name, data):
     template = Path(
         f'{current_app.config.get("TEMPLATE_PATH")}/{template_name}.xml'
     ).read_text()
-    jnja_template = Template(template, autoescape=True)
+    jnja_template = Template(template, autoescape=False)
```

Comment:
Before:
![image](https://github.com/user-attachments/assets/682f66fc-ace8-4aad-8a7b-689545e26b92)

Now:
![image](https://github.com/user-attachments/assets/2ae165fa-48e3-4414-bd0c-6306bb20025a)

Reply:
I am surprised no one complained about this before.